plugins/rollbar_dashboards/app/views/rollbar_dashboards/_project.html.erb

```diff
@@ -0,0 +1,18 @@
+<%
+  if project.rollbar_dashboards_settings.any?
+    @dashboard_data = project.rollbar_dashboards_settings.map do |dashboard|
+      items = Rails.cache.fetch(['project', dashboard], expires_in: 10.minutes) do
+        RollbarDashboards::Client.new(dashboard).top_errors&.first(RollbarDashboards::Setting::NUM_ITEMS)
```

Comment:
this could potentially cache nil ... which means the cache won't work ...
might be best to move this somewhere in the helper/model/client code to properly test it too

Reply:
possibly also good to make this a lazy-load so the page stays fast and everything has the same workflow